src/content/docs/pt-br/basics/astro-components.mdx

```diff
@@ -51,19 +51,19 @@ const dados = await fetch('ALGUMA_URL_SECRETA_API/usuarios').then(r => r.json())
 <!-- Seu template está aqui! -->
 ```
 
-A cerca de código é projetada para garantir que o JavaScript que você escreve nela está "cercado". Ele não irá escapar para a sua aplicação frontend ou cair na mão dos seus usuários. Você pode com segurança escrever aqui código que é custoso ou sensível (como uma chamada ao seu banco de dados privado) sem se preocupar com ele cair no navegador do seu usuário.
+A cerca de código é projetada para garantir que o JavaScript que você escreve nela está "cercado". Ele não irá escapar para a sua aplicação frontend ou cair nas mãos dos seus usuários. Você pode com segurança escrever aqui código que é custoso ou sensível (como uma chamada ao seu banco de dados privado) sem se preocupar com ele indo parar no navegador do seu usuário em algum momento.
 
 :::tip
 Você pode até mesmo escrever TypeScript no script do componente!
 :::
 
 ### O Template do Componente
 
-O template do componente está abaixo da cerca do código e determina o HTML que irá sair do seu componente.
+O template do componente está abaixo da cerca do código e determina o HTML que será produzido pelo seu componente.
 
-Se você escrever puro HTML aqui, o seu componente irá renderizar esse HTML em qualquer página Astro em que é importado e utilizado.
+Se você escrever HTML puro aqui, o seu componente irá renderizar esse HTML em qualquer página Astro em que ele é importado e utilizado.
```

Comment:
Reverse order of adjective and noun for a more natural structure; added the extra anaphoric pronoun "ele" because the sentence goes on for too long and I felt like we were starting to lose some cohesion - pronoun glues everything back together.